packages/ui/src/forum/components/category/CategoryListItem.tsx

```diff
@@ -93,9 +75,12 @@ const CategoryListItemTitle = styled.h5`
 
 export const CategoryListItemStyles = styled(TableListItem)`
```

Comment:
This component shouldn't extends `TableListItem` anymore because it's no longer the same kind of component (also it creates a slight misalignment of the first item). Instead maybe some style from `ThreadItemWrapper` could be factored out and reused here